.github/workflows/test.yml

```diff
@@ -150,12 +150,12 @@ jobs:
       - name: filter out DONTCOVER
         # yamllint disable
         run: |
-          excludelist="$(find ./ -type f -name '*.go' | xargs grep -l 'DONTCOVER')"
-          excludelist+=" $(find ./ -type f -name '*.pb.go')"
-          excludelist+=" $(find ./ -type f -name '*.pb.gw.go')"
-          excludelist+=" $(find ./ -type f -path './tests/mocks/*.go')"
+          excludelist="$(find . -type f -name '*.go' | xargs grep -l 'DONTCOVER')\n"
+          excludelist+="$(find . -type f -name '*.pb.go')\n"
+          excludelist+="$(find . -type f -name '*.pb.gw.go')\n"
+          excludelist+="$(find . -type f -path './tests/mocks/*.go')"
           for filename in ${excludelist}; do
-            filename=$(echo $filename | sed 's/^./github.com\/cosmos\/cosmos-sdk/g')
+            filename=$(echo $filename | sed 's/^./github.com\/celestiaorg\/celestia-app/g')
```

Comment:
[Blocking]
If I understand right, the purpose of this PR is not to upload the proto generated files, etc, to test coverage.
I created a dummy branch to check which files exist at this step:
https://github.com/celestiaorg/celestia-app/actions/runs/3413868856/jobs/5681165683

As you can see, the tree doesn't contain any folders named `github.com`. 

Also, the `excludelist` contains the right files, as  per:
```
          excludelist="$(find . -type f -name '*.go' | xargs grep -l 'DONTCOVER')\n"
          excludelist+="$(find . -type f -name '*.pb.go')\n"
          excludelist+="$(find . -type f -name '*.pb.gw.go')\n"
          excludelist+="$(find . -type f -path './tests/mocks/*.go')"
```
Thus, I guess  this could be updated to the following:

```
          for filename in ${excludelist}; do
            echo "Excluding ${filename} from coverage report..."
            sed -i.bak "/$(echo $filename | sed 's/\//\\\//g')/d" coverage.txt
          done
```
Let me know if I am missing something please.

Reply:
I think the coverage file does contain files with github prefix, but it shouldn't matter I think, have removed it

Reply:
I'll double check by triggering the CI once more

Reply:
Works